undebt/pattern/interface.py

```diff
@@ -74,8 +74,19 @@ def patterns_from_files(pattern_files):
 
 def load_module(path):
     """Loads a module from its path."""
-    pattern_name = os.path.splitext(os.path.basename(path))[0]
-    return imp.load_source(pattern_name, path)
+    return __import__(maybe_path_to_module_name(path), fromlist=[''])
+
+
+def maybe_path_to_module_name(maybe_path):
+    relpath = os.path.relpath(maybe_path)
+    if relpath.startswith('..'):
+        raise ValueError("Relative imports not allowed: {}", relpath)
```

Comment:
2 things:
- "relative imports" means something else, I believe you mean relative file paths
- this isn't going to do the string formatting you expect:

```
>>> relpath = '../foo'
>>> raise ValueError("Relative imports not allowed: {}", relpath)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: ('Relative imports not allowed: {}', '../foo')
```